app/views/admins/_user_entry.mobile.haml

```diff
@@ -0,0 +1,70 @@
+
+%li.user.media
+  %div.pull-left
+    - if user.person
+      %span.media-object
+        = person_image_tag(user.person, size: :thumb_small)
+
+  %div.media-body.row
+    %div.pull-right
+      %span.label.label-default
+        = t('.id')
+        = user.id
+      %span.label.label-info
+        = t('.guid')
+        = user.person.guid if user.person
+
+    %h4.media-heading
+      = user.person.name if user.person
+
+    %div
+      .unstyled.text-right.actions
+        = link_to t('admins.user_search.view_profile'), person_path(user.person), class: 'btn btn-default btn-block'
+        = link_to t('admins.user_search.add_invites'), add_invites_path(user.invitation_code), class: 'btn btn-info btn-block'
```

Comment:
- Line is too long. [126/120]
- Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.